packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -127,13 +128,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setRootView:(UIView *)rootView toRootViewController:(UIViewController *)rootViewController;
 
-/// This method controls whether the App will use RuntimeScheduler. Only applicable in the legacy architecture.
-///
-/// @return: `YES` to use RuntimeScheduler, `NO` to use JavaScript scheduler. The default value is `YES`.
-- (BOOL)runtimeSchedulerEnabled;
-
-@property (nonatomic, strong) RCTSurfacePresenterBridgeAdapter *bridgeAdapter;
```

Comment:
For the same reason why I think we should keep the Bridge, we should probably keep this as well. I don't know if there are libs or pieces of code that tries to access this property by reaching for the AppDelegate.